### DISCUSSION TITLE
Cannot find the Ranking degree

### DISCUSSION THREAD

**Discussion by marino-mrc** [gephi/gephi#2523] [Q&A] [answered]
Hi,
I'm using gephi 0.9.5 on Ubuntu 20.04 LTS but I cannot find the ranking table.
Clicking on "Data Laboratory" shows a table containing the nodes but It's not clear to me how can I add the Degree.
Any help is highly appreciated!

Thank you.
Regards,
Marco

**Comment by mbastian** [comment, marked as answer]
Hi Marco, go to Statistics > Degree and after you've run that successfully the column will be there in the Data Laboratory.
